flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBMemoryControllerUtils.java

```diff
@@ -168,4 +170,26 @@ static long calculateRocksDBMutableLimit(long bufferSize) {
     static boolean validateArenaBlockSize(long arenaBlockSize, long mutableLimit) {
         return arenaBlockSize <= mutableLimit;
     }
+
+    /** Factory for Write Buffer Manager and Bock Cache. */
+    public interface RocksDBMemoryFactory extends Serializable {
+        Cache createCache(long cacheCapacity, double highPriorityPoolRatio);
+
+        WriteBufferManager createWriteBufferManager(long writeBufferManagerCapacity, Cache cache);
+
+        RocksDBMemoryFactory DEFAULT =
+                new RocksDBMemoryFactory() {
+                    @Override
+                    public Cache createCache(long cacheCapacity, double highPriorityPoolRatio) {
+                        // TODO use strict capacity limit until FLINK-15532 resolved
+                        return new LRUCache(cacheCapacity, -1, false, highPriorityPoolRatio);
```

Comment:
Any reason not to just call the existing static method `RocksDBMemoryControllerUtils.createCache`?

Reply:
No good reason, I'll change this method to call `RocksDBMemoryControllerUtils.createCache`. Thanks.